src/layouts/AppBar/AppBar.component.tsx

```
@@ -0,0 +1,81 @@
+                {isDesktop ? (
+                    <Stack direction="row" spacing={8} alignItems="center">
+                        <Link component="a" href="/" lineHeight={0}>
+                            <Paper
+                                component="img"
+                                alt="Themesberg Logo"
+                                src="/assets/images/logo/logo.svg"
+                                sx={{
+                                    border: 'none',
+                                    backgroundColor: 'background.default',
+                                }}
+                            ></Paper>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)